apis/router/src/SwapRequestStatistics.ts

```diff
@@ -160,7 +160,7 @@ export class SwapRequestStatistics {
 
   requestRejected(reason: ResponseRejectReason) {
     ++this.rejected
-    ++this.rejects[reason]
+    this.rejects[reason] = (this.rejects[reason] || 0) + 1
```

Comment:
all reasons are set to 0 already in constructor so this seems totally uneccasary

Reply:
doesn't really matter as long as ts is happy